### DISCUSSION TITLE
Sonoff  with tasmota-sensor.bin

### DISCUSSION THREAD

**Discussion by novembre62** [arendst/Tasmota#20005] [Support request]
On sonoff basic and tasmota-sensor.bin (13.2.0) the tasmota wifi network appears, but it is not possible to connect for the router SSID and password configuration......while using tasmota.bin, the connection occurs normally.  How can I solve it.  Thank you

**Comment by sfromis** [comment]
By not getting the most crappy device, and expecting it to be a platform to extend on.

(You could also try doing a minimalistic custom build only having features you need)

**Comment by novembre62** [comment]
I need to read some INA219 sensors and it seems to me that only tasmota-sensor.bin has the necessary drivers. Unfortunately I don't have the knowledge to create a custom build, but if I could configure tasmota-sensor with the router it would be simple

**Reply by sfromis** [reply]
Compiling is not as hard as you think. https://tasmota.github.io/docs/Compile-your-build/

But my primary response is that the Sonoff Basic is just too crappy a device to be used as a platform to add stuff to.

**Comment by novembre62** [comment]
I solved it with tasmota-sensor.bin 9.1.0

tasmota-sensor.bin 13.2.0 does not accept connection to its wifi network for configuration with the router

**Reply by sfromis** [reply]
No, it is your crappy hardware which does not work well, with current functioning Tasmota.

**Comment by novembre62** [comment]
with tasmota.bin 13.2.0 it works with the same hardware... why?

**Reply by sfromis** [reply]
Because your hardware is crap. It can borderline power itself, but just a little bit more power draw with more Tasmota features crashes it.

> (You could also try doing a minimalistic custom build only having features you need)

**Comment by novembre62** [comment]
what hardware would be good, for ina219 and tasmota ?

**Reply by sfromis** [reply]
My preference is a dev board, with a good voltage regulator, like in the NodeMCU form factor (many others exist too). Beware that D1 Mini clones sold "typically" do not have a good voltage regulator.

**Comment by novembre62** [comment]
thanks for your advice
